lib/node_modules/@stdlib/stats/base/smediansorted/lib/ndarray.js

```diff
@@ -21,7 +21,7 @@
 // MODULES //
 
 var float64ToFloat32 = require( '@stdlib/number/float64/base/to-float32' );
-var floor = require( '@stdlib/math/base/special/floor' );
+var floorf = require( '@stdlib/math/base/special/floorf' );
```

Comment:
```suggestion
var floor = require( '@stdlib/math/base/special/floor' );
```

This change should be reverted. It is used to round `N / 2`, which is a double, not a float. For typed arrays, N may be up to 2**53, which exceeds the maximum safe float32 integer.

Reply:
I'll keep this in mind!